Sources/Sentry/Public/SentryOptions.h

```diff
@@ -8,6 +8,20 @@ NS_ASSUME_NONNULL_BEGIN
 NS_SWIFT_NAME(Options)
 @interface SentryOptions : NSObject
 
+/**
+ * Init SentryOptions.
+ * @param options Options dictionary
+ * @return SentryOptions
+ */
+- (_Nullable instancetype)initWithDict:(NSDictionary<NSString *, id> *)options
+                      didFailWithError:(NSError *_Nullable *_Nullable)error;
+
+/**
+ * Optional, custom cache directory. Use when default one can not be accessed, e.g. in security
+ * environment.
+ */
+@property (nonatomic, strong) NSString *_Nullable cacheDirectory;
```

Comment:
`h`: Please rename the property to `cacheDirectoryPath` to reflect in the naming it's a path. We can also make it nonnullable, as it has to have a value.

```suggestion
@property (nonatomic, copy) NSString *cacheDirectoryPath;
```